frontend/packages/citizen-frontend/src/localization/sv.tsx

```diff
@@ -702,7 +687,7 @@ const sv: Translations = {
         units: {
           title: 'Ansökningsönskemål',
           startDateMissing:
-            'För att välja önskade enheter, välj det första önskade startdatumet i avsnittet om "Servicebehov"',
+            'För att kunna välja önskade enheter, välj först det önskade startdatumet i avsnittet om "Servicebehov"',
```

Comment:
Do we actually have a section called "Servicebehov"? (I think we used to, but not anymore)